code/game/machinery/OpTable.dm

```diff
@@ -82,17 +82,19 @@
 	if(!victim || !victim.lying || victim.loc != loc)
 		suppressing = FALSE
 		victim = null
-	var/mob/living/carbon/human/H = locate() in loc
-	if(istype(H))
-		if(H.lying)
-			icon_state = H.pulse() ? "[modify_state]-active" : "[modify_state]-idle"
-			victim = H
-			if(H.stat == DEAD || H.is_asystole() || H.status_flags & FAKEDEATH)
-				icon_state = "[modify_state]-critical"
+		var/mob/living/carbon/human/H = locate() in loc
+		if(istype(H))
+			if(H.lying)
+				icon_state = H.pulse() ? "[modify_state]-active" : "[modify_state]-idle"
+				victim = H
 	if(victim && !victim.isSynthetic())
 		if(suppressing && victim.sleeping < 3)
 			victim.Sleeping(3 - victim.sleeping)
 			victim.willfully_sleeping = FALSE
+		if(ishuman(victim))
```

Comment:
Victims cannot be non-human, so I don't think we need this line.

Rest is good though.

Reply:
Unfortunately removing this line causes the operating table icon to not update. I do not know why